src/main/java/com/google/cloud/resourcemanager/LienInfo.java

```diff
@@ -0,0 +1,222 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.resourcemanager;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+import com.google.common.base.Function;
+import java.io.Serializable;
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * A Google Cloud Resource Manager lien metadata object.A Lien represents an encumbrance on the
+ * actions that can be performed on a resource.
+ */
+public class LienInfo implements Serializable {
+
+  static final Function<com.google.api.services.cloudresourcemanager.model.Lien, LienInfo>
```

Comment:
Why are there two Lien classes in different packages. Can we avoid that?

Reply:
1.`com.google.api.services.cloudresourcemanager.model.Lien` is the Java data model class and that specifies how to parse/serialize into the `JSON` that is transmitted over HTTP.

2.`com.google.cloud.resourcemanager.Lien` is layer of that model.

Reply:
Perhaps the second could be named LienLayer?

Reply:
I think we should go with Lien instead of LienLayer because Api itself return the instance of Lien
see https://cloud.google.com/resource-manager/reference/rest/v1/liens/create#response-body

Reply:
I think we should go with Lien instead of LienLayer because Api itself return the instance of Lien
see https://cloud.google.com/resource-manager/reference/rest/v1/liens/create#response-body

Reply:
I just figured it would be easier to rename this class than the other one; but either way reusing the name for two different classes like this will confuse and irritate developers.

Reply:
done